json/README.md

```diff
@@ -2,7 +2,34 @@
   ~ Copyright (c) VMware, Inc. 2023. All rights reserved.
   ~ SPDX-License-Identifier: Apache-2.0
   -->
-## To run the JSON example
+
+The new JSON related APIs allow you to convert a JSON String into a binary form that can be stored in a GemFire region. 
+This example starts a GemFire cluster with one locator and two servers with a replicated region called `example-region`.
+It then uses the JSON related APIs to create a `JsonDocumentFactory` which converts a JSON string into a JsonDocument.
```

Comment:
Shall we add supported PDX and BSON formats somewhere in the README. It was mentioned in the example code. I think it will be good to add which format is suitable for json with schema or not.